resolvers/webpack/index.js

```diff
@@ -265,7 +266,7 @@ function findExternal(source, externals, context) {
 
   // else, vanilla object
   for (var key in externals) {
-    if (!externals.hasOwnProperty(key)) continue
+    if (!has(externals, key)) continue
```

Comment:
The first one, I also don't know why someone *would* do it, but given that it's a possibility, that would be a really silly reason for this package to break, don't you think?

As to the second, for a tool like an eslint plugin, that's a stronger case, since it won't be running in browser environments, or environments with unknown third-party code.

I'll remove the dependency if you insist, but I think this is still strictly better because you don't have to think about all these edge cases. I do not think `in` or `hasOwnProperty` are well-understood idioms, which is why this problem keeps coming up in libs all over the place, and why `has` exists.